tests/phpunit/tests/webfonts/wpThemeJsonWebfontsHandler.php

```diff
@@ -131,6 +131,7 @@ private function setup_theme_and_test( $theme_name ) {
 		switch_theme( $theme_name );
 		do_action( 'after_setup_theme' );
 		wp_clean_theme_json_cache();
+		do_action( 'plugins_loaded' );
```

Comment:
Is this necessary as we are checking the theme JSON?

Reply:
@mukeshpanchal27 https://github.com/WordPress/wordpress-develop/blob/39bfc2580d9b0ea7e6ff45b8d904408d925cbc4b/src/wp-includes/default-filters.php#L354